src/main/webapp/js/wikiSearch.js

```diff
@@ -56,17 +56,21 @@ function displayOtherOptions(){
   var options = keyword.split(" ");
   var displayList = document.getElementById('display-list');
   options.forEach(function(opt) {
-    var link = document.createElement('a');
-    var text = document.createTextNode(opt);
-    link.appendChild(text);
-    link.href = "/info-present.html/?kw=" + opt;
```

Comment:
The problem with CSS not applying properly was caused by misplaced slashes in various places.

Let's take a look at the header of `info-present.html`, it has this line:

```
<link rel="stylesheet" href="css/info-present.css">
```
The lack of leading slash in `href` means that the link is relative. Basically, the browser will add the value of `href` after the last slash in the URL. When you were looking at `http://localhost:8080/info-present.html`, the browser just removed the name of the file (`info-present.html`) and stuck `css/info-present.css` there, which turned the URL of the resource into `http://localhost:8080/css/info-present.css`. It downloaded the file and it was fine.

Now, when you were creating these disambiguation links, if you look at your `href`, it has a slash at the end, so when you clicked on the link, you went to `http://localhost:8080/info-present.html/?kw=mini+starbucks`. To get the link for CSS, the browser removed everything after the last slash and stuck `css/info-present.css` there, thus getting `http://localhost:8080/info-present.html/css/info-present.css`. When it tried to download that CSS file, it got `404 - Not Found` and thus you saw no style.

I'd say that the main problem is the relative link in stylesheet reference, it needs to have a leading slash:

```
<link rel="stylesheet" href="/css/info-present.css">
```
If you have opened Developer Tools in the browser you would see the error there, saying that `.../info-present.html/css/info-present.css` was not found -- this was the hint.